mkt/zadmin/templates/admin/base.html

```diff
@@ -73,15 +73,6 @@
                       </li>
                     </ul>
                   </li>
-                {% else %}
```

Comment:
I tried loading `/admin/` without being logged on, and I ended up on a login page. I don't think this link even worked correctly.